style2.css

```diff
@@ -0,0 +1,294 @@
+@media screen and (max-width: 1440px) {
+	
+	
+}
+
+@media screen and (max-width: 1120px) {
+	.header__menu-link {
+		font-size: 14px;
+	}
+}
+@media screen and (max-width: 1090px) {
+.header__menu-link {
+	font-size: 13px;
+}
+}
+
+@media screen  and (max-width: 1062px) {
+.menu{
+	display: none;
+}
+.header__menu-link {
+	display: none;
+}
+}
+
+
+
+@media screen and (max-width: 1024px) {
+.body {
+text-align: center;
+align-items: center;
+}
+
+.header {
+diplay: flex;
+padding: 0;
+margin: 0 auto;
+}
+
+
+.main {
+padding: 0;
+margin: 0 auto;
+	
+}
+
+
+.header__menu {
+display: none;
+}
+
+.header__menu-link {
+	display: none;
+
+}
+.header__line {
+	display: none;
+}
+
+
+.main__title {
+	font-size: 90px;
+	margin-bottom: 10px;
+	
+}
+.main__subtitle {
+	font: 15px 'montserratmedium';
+	margin-bottom: 0;
+	padding: 0;
+	text-align: center;
+}
+
+.main__info {
+display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+}
+
+
+
+.main__content {
+	margin: 20px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/tesla394/pullRequest?prid=5642156)